hover/effects/icon-transitions/_icon-setup.sass

```diff
@@ -0,0 +1,18 @@
+// Mixin to position icons
+// This mixin is used in every icon-transition mixin and must be included
+// if you are using any other icon-transition files
+
+@mixin icon-setup($icon-size: 20px, $position: right, $spacing-inner: 10px, $spacing-outer: 15px)
+    position: relative
+    @if $position == right
+        padding:
+            right: $icon-size + $spacing-outer + $spacing-outer
+    @else
+        padding:
+            left: $icon-size + $spacing-outer + $spacing-outer
```

Comment:
`$icon-size + ($spacing-outer * 2)` here and above. Can assign to a variable to be DRY too